rootfs/standard/var/www/mynode/static/css/mynode_dark.css

```diff
@@ -30,7 +30,8 @@ a:active {
 .settings_block_subheader,
 .settings_block,
 table,
-#cli_contents {
+#cli_contents,
```

Comment:
I'd prefer these to be classes as opposed to specific ID references in case they need to be re-used.

Reply:
I didn't introduce #cli_contents
but it's only used in bitcoin-cli page, so it's not a big deal

Reply:
Ha, OK, then I'm complaining about my own shortcuts and hacks. ;-)

Reply:
Does this also need to be made in the non-dark version? This is why I was concerned about two CSS files.

Reply:
I'm not sure if I understand you.
small_message class (which shows the witty quotes) has the default black color in the non-dark version. I'm just changing its color to off-white.

Reply:
#cli_contents is for displaying the output of CLI commands and few javascript functions are attached to it.